src/ThreeEditor/components/Dialog/OpenFileDialog.tsx

```diff
@@ -110,6 +111,7 @@ export function OpenFileDialog(props: OpenFileProps) {
 								))}
 							</List>
 							<Button
+								aria-labelledby='Load example button'
```

Comment:
Did you mean?
```suggestion
								aria-label='Load example button'
```

Reply:
yes, looks like I misunderstood how to use aria-labelledby